.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Current File",
+            "type": "python",
+            "request": "launch",
+            "program": "chessington/app.py",
+            "console": "integratedTerminal"
+        },
+        {
+            "name": "Python: Current File",

```

Comment:
This section won't be necessary for some learners, but should prevent those who see the [duplicate env](https://github.com/microsoft/vscode-python/issues/10847) bug from having issues